static/js/tools/stat_var/info.tsx

```diff
@@ -32,6 +32,29 @@ class Info extends Component {
           thousands of statistical variables to choose from, arranged in a
           topical hierarchy.
         </p>
+        <p>
+          Or you can start your exploration with these statistical variables ...
+        </p>
+        <ul>
+          <li>
+            <a href="#statVar=Count_Person">Population</a>
+          </li>
+          <li>
+            <a href="#statVar=CumulativeCount_Vaccine_COVID_19_Administered">
+              Cumulative Count of COVID_19, Vaccine Administered
+            </a>
+          </li>
+          <li>
+            <a href="#statVar=Daily_PrecipitationRate_RCP85">
+              Daily Precipitation Rate Based on RCP 8.5
+            </a>
+          </li>
+          <li>
+            <a href="#statVar=Annual_Generation_Energy_NonRenewableWaste">
+              Annual Generation of Non Renewable Waste
```

Comment:
hm that stat var seems problematic (e.g. the word energy is missing from the sv name)

how about one of these instead: https://autopush.datacommons.org/tools/stat-var#statVar=Annual_Consumption_Electricity
https://autopush.datacommons.org/tools/stat-var#statVar=Annual_Imports_Electricity

and maybe an economics sv:
https://autopush.datacommons.org/tools/stat-var#statVar=GiniIndex_EconomicActivity

Reply:
added these